app/views/handlers/block_factory.py

```diff
@@ -63,7 +64,7 @@ def get_block_handler(
 
     section_id = schema.get_section_id_for_block_id(block_id)
 
-    if to_list_item_id:
+    if to_list_item_id or block_type == "UnrelatedQuestion":
```

Comment:
I did initially try this but it doesn't work because the block can be `RelationshipCollector` without a `list_item_id` when you first direct to this block. After the collector block handler is initialised, we redirect it to the relationship route.

https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/app/routes/questionnaire.py#L188-L189

Reply:
Fair enough.